python/xi_plugin/host.py

```diff
@@ -99,6 +98,17 @@ def shutdown(self, peer, **params):
         peer.done = True
         self.plugin.shutdown()
 
+    def other_cmd(self, peer, method, params):
+        '''Unknown RPC, presumably a custom command provided by the plugin.'''
```

Comment:
I think I'd prefer a design where the RPC to the plugin was a wrapper method like "custom_command" so that they are distinct from defined parts of the plugin API, and plugins should ignore top-level commands they don't know about. That way it would be possible to add new plugin notification RPCs in a backwards compatible way. I also think the separation more clearly distinguishes two different things: RPCs defined by the plugin, and RPCs defined by the host.